x/participation/simulation/simap.go

```diff
@@ -0,0 +1,15 @@
+package simulation
```

Comment:
rename file to `simapp.go` later at some point

Reply:
can do now if needed. you'll have to reapprove though

Reply:
@aljo242